wurst/objects/abilities/beastmaster/Druid/NaturesBond.wurst

```diff
@@ -0,0 +1,82 @@
+package NaturesBond
+
+// Standard Library Imports:
+import Icons
+import ChannelAbilityPreset
+import BuffObjEditing
+import BuffIds
+import Abilities
+
+// Local Imports:
+import LocalObjectIDs
+import ToolTipsUtils
+import ColorUtils
+import HealingSystem
+import Pets
+
+
+let COOLDOWN = 60.
+let MANA_COST = 20
+let HEAL_DIST = .5
+let DURATION = 15.
+let CAST_RANGE = 15
+let BUTTON_POS_X = 3
+let BUTTON_POS_Y = 2
+let HOTKEY = "D"
+let AOE = 4000
+let NAME = "Nature's Bond"
+let ICON = Icons.bTNSpiritLink
+let TOOLTIP_BASIC = "Nature's Bond"
+let TOOLTIP_EXT   = "The strong bond between the Druid and his pet causes them to split all damage between themselves for "
+    + DURATION.toToolTipOrange() + " seconds. They also share " + (HEAL_DIST*100).toToolTipGreen() + " %".color(HEALTH_COLOR)
```

Comment:
Is there a reason for having the `%` character colored differently? All other % values mentioned in tooltips share the same color as the value, you can do `(HEAL_DIST).toToolTipGreen()`, it will format the string to display "50%"